hub/powertoys/run.md

```diff
@@ -160,21 +165,21 @@ The PowerToys Run calculator plugin supports the following operations:
 | Division |  a / b | |
 | Modulo/Remainder | a % b | |
 | Exponentiation | a ^ b | |
-| Ceil | ceil( x.y ) |  Rounds a number up to the next larger integer. |
-| Floor | floor( x.y ) | Rounds a number down to the next smaller integer. |
+| Ceiling function | ceil( x.y ) |  Rounds a number up to the next larger integer. |
+| Floor function | floor( x.y ) | Rounds a number down to the next smaller integer. |
+| Rounding | round( x.abcd ) | Rounds to the nearest integer.<br />_(Example: `round(8.7867)`)_ |
 | Exponential function | exp( x ) | Returns e raised to the specified power. |
 | Maximum | max( x, y, z ) | |
-| Minimum | max( x, y, z ) | |
-| Absolute | abs( - x ) | Absolute value of a number |
-| Log10 | log( x ) | |
-| Log base e | ln( x ) | |
+| Minimum | min( x, y, z ) | |
+| Absolute | abs( -x ) | Absolute value of a number. |
+| Logarithm base 10 | log( x ) | |
+| Logarithm base e | ln( x ) | |
 | Square root | sqrt( x ) | |
-| Power | pow( x, y ) | Calculate a number raised to the power of some other number. |
-| Factorial | x ! | |
-| Sign | sign( - x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
-| Round | round( x.abcd ) | Example: `round(8.7867)` |
-| Random | rand() | |
-| Pi | +pi | Returns the number of Pi. |
+| Power of x | pow( x, y ) | Calculate a number (x) raised to the power of some other number (y). |
+| Factorial | x! | |
+| Sign | sign( -x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
+| Random number | rand() | Returns a fractional number between 0 and 1. |
+| Pi | pi | Returns the number of Pi.<br />_(To simply display pi, a sign is required like in this example: `+pi`.)_ |
```

Comment:
Small thing: "pi" doesn't need a capital P. It's a letter, not a given name. And I think "the value of pi" sounds better, what do you think?

Reply:
I think it looks strang if we start with lower case letter as other start uppercase.

I think it's okay as is. In this case Pi is the name of the number and therefore we can handle it as such one.

Reply:
Or... we could make all the "actions" lowercase...

Reply:
let's keep as is